### DISCUSSION TITLE
Add fetching / pushing of tags to `Fetch upstream` button

### DISCUSSION THREAD

**Discussion by olifre** [community/community#14330] [Other Feature Feedback, Questions, & Ideas]
The `Fetch upstream` button is extremely useful to keep forks up to date. 

However, it does not fetch / replicate tags from upstream in the fork, which would be useful to replicate tagged states / releases. 
This still has to be done by manually checking out the repository, fetching all tags from upstream and pushing all tags into the fork. 

It would be nice if the `Fetch upstream` button could do that automatically.

**Comment by 3nids** [comment]
yes please!

**Comment by mbernasocchi** [comment]
That would help many downstream projects
